docs/higlass_server.rst

```diff
@@ -115,12 +115,30 @@ To filter by a specific filetype, use the `t=filetype` parameter:
 
     curl localhost:8000/api/v1/tilesets?t=cooler
 
-To fileter by datatype, use the `dt=datatype` parameter:
+To filter by datatype, use the `dt=datatype` parameter:
 
 .. code-block:: bash
 
     curl localhost:8000/api/v1/tilesets?dt=matrix
 
+Retrieving properties of a tileset, for a specific `uuid`:
+
+.. code-block:: bash
+
+    curl localhost:8000/api/v1/tilesets/${uuid}/
+    
+To delete a tileset, specify the tileset `uuid` in the URL, and use the `DELETE` method with authentication credentials:
+
+.. code-block:: bash
+
+    curl --user ${username}:${password} --request DELETE http://localhost:8000/api/v1/tilesets/${uuid}/
+    
+To modify a tileset's name, specify the tileset `uuid` in the URL, use the `PUT` method with authentication credentials, and specify the new name in the JSON object passed to the request:
```

Comment:
Make it explicit that this *only* modifies the name and not any other fields.

Reply:
The example demonstrating the use of `PATCH` to modify a tileset is limited strictly to changing the name.